src/chapters/setup-intro-data-types/studio.rst

```diff
@@ -1,2 +1,68 @@
 Studio: Data Types
 ===================
+
+Get cosy with Java syntax by writing a console program that calculates the 
+area of a circle based on input from the user.
+
+Creating your class
```

Comment:
Just a general note, we should try to remain as consistent as possible on a page about using "we" or "you" in the wording.  We want to avoid switching the subject focus mid-stream.

EX: 
1) "We love to code, but you need to be careful about..."  (Nope!)
2) "We love to code, but we need to be careful about..."  (OK)
3) "You love to code, but we need to be careful about..."  (Nope!)
4) Etc.